model/src/main/kotlin/ScanResultContainer.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model
+
+/**
+ * A container for [ScanResult]s for the package identified by [id].
```

Comment:
Maybe "for [ScanResult]s from different scanners"? Or do we explicitly want to allow things like multiple scan results from the same scanner, but with different configurations?

Reply:
Yes, e.g. results from the same scanner but with different provenance should be possible.

Reply:
I was more thinking about different scanner details / scanner command line options, but for the same provenance. I guess we'd want to support that, too?